apps/labs/posts/ruby-wrappers-for-the-xnd-project.md

```diff
@@ -0,0 +1,416 @@
+---
+title: 'Ruby wrappers for the XND project'
+published: September 15, 2019
+author: sameer-deshmukh
+description: 'This post is a rather long and detailed introduction to the XND ruby wrapper. There will also be some details on the implementation of the wrapper and how it differs from the Python wrapper (which existed before the Ruby wrapper).'
+category: [Developer workflows, PyData ecosystem]
+featuredImage:
+  src: /posts/ruby-wrappers-for-the-xnd-project/blog_feature_var1.svg
+  alt: 'An illustration of a brown and a dark brown hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/ruby-wrappers-for-the-xnd-project/blog_hero_var1.svg
+  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+---
+
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-generate-toc again  -->
```

Comment:
@gabalafou I was trying to make the markdown comment work (they were visible before). Any idea what's the right flavor of comments to use here?
If not, I suggest we remove them and merge.